dependencies/tools/prune-mathjax

```diff
@@ -55,6 +53,28 @@ rm -rf $MATHJAX_DIR/fonts/HTML-CSS/TeX/svg
 rm -rf $MATHJAX_DIR/fonts/HTML-CSS/TeX/png
 rm -rf $MATHJAX_DIR/fonts/HTML-CSS/TeX/eot
 rm -rf $MATHJAX_DIR/fonts/HTML-CSS/TeX/woff
+rm -rf $MATHJAX_DIR/fonts/HTML-CSS/STIX-Web
```

Comment:
How did you decide which fonts to blow away vs keep here? Just curious, don't know much about how we use MathJax.

Reply:
The additions here are new fonts in MathJax 2.7 that didn't exist in MathJax 2.6, so decided to keep only those fonts that we already ship. The fonts are quite large and prior to the additional pruning we would add ~30mb to RStudio's size. (Even afterwards, the zipfile is up by ~2mb and our download will reflect that.)